torchvision/prototype/datasets/_builtin/food101.py

```diff
@@ -0,0 +1,111 @@
+import enum
+import functools
+from pathlib import Path
+from typing import Any, Tuple, List, Dict, cast, Optional, BinaryIO
+
+from torchdata.datapipes.iter import IterDataPipe, Filter, Mapper, CSVParser, JsonParser, Demultiplexer
+from torchvision.prototype.datasets.utils import Dataset, DatasetInfo, DatasetConfig, HttpResource, OnlineResource
+from torchvision.prototype.datasets.utils._internal import (
+    hint_shuffling,
+    hint_sharding,
+    path_comparator,
+    getitem,
+    INFINITE_BUFFER_SIZE,
+)
+from torchvision.prototype.features import Label, EncodedImage
+
+
+class Food101Demux(enum.IntEnum):
+    METADATA = 0
+    IMAGES = 1
+    EXTRA = 2
```

Comment:
Do we actually need this enum? IIUC, we only use it once to classify the archive. If that is the case, this is unneeded overhead. Just hardcode `0` and `1` there. Enums are useful if we need to have access to hardcoded values multiple times.